insitu/tests/test_data_responsible.py

```diff
@@ -95,11 +95,42 @@ def test_edit_network_responsible(self):
         network = base.DataResponsibleFactory(is_network=True)
         resp = self.client.post(reverse('responsible:edit_network',
                                        kwargs={'pk': network.pk}),
-                               data)
+                                data)
         self.assertEqual(resp.status_code, 302)
         data['networks'] = []
         self.check_single_object(models.DataResponsible, data)
 
+    def test_edit_network_members_responsible(self):
+        data = self._DATA
```

Comment:
You should send here a list of members, a dict like {"members": [1, 2, 3]}, where 1,2,3 are pk's.